### PR TITLE
Install Kargo

### PR DESCRIPTION
**Pull request by dtzar**
## Purpose
Install Kargo

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: gitops/environments/default/addons/kargo/values.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+api:
+  adminAccount:
+    passwordHash: '$2a$10$2PqnGA40RswbHnvw5TKV2OhxtCIVaygvSHX7TUv8ADBF3bGD2vhN2'
+    tokenTTL: "24h"
+    tokenSigningKey: "iwishtowashmyirishwristwatch"
+
+controller:
+  logLevel: "DEBUG"
```
Comment (nawazkh) [inline gitops/environments/default/addons/kargo/values.yaml:8]: nit: new line

File: gitops/bootstrap/control-plane/addons/oss/addons-kargo-appset.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
```
Comment (nawazkh) [inline gitops/bootstrap/control-plane/addons/oss/addons-kargo-appset.yaml:1]: New eyes 👀 
Can we add more context on why we are adding this file for my sanity ? 
To me it looks like we are trying to install KArgo. But is that it?

Comment (dtzar) [reply gitops/bootstrap/control-plane/addons/oss/addons-kargo-appset.yaml:1]: Yes

**Review by nawazkh — APPROVED** [review]
🚀

**Pull request merged**